src/HotChocolate/ApolloFederation/src/ApolloFederation/Extensions/SchemaBuilderExtensions.cs

```diff
@@ -0,0 +1,18 @@
+namespace HotChocolate.ApolloFederation.Extensions
+{
+    public static class SchemaBuilderExtensions
+    {
+        public static ISchemaBuilder AddApolloFederation(
+            this ISchemaBuilder builder)
+        {
+            builder.AddType<EntityType>();
```

Comment:
since it is a public extension we need to check the builder for null.

Reply:
Check added